py/kubeflow/testing/util.py

```diff
@@ -448,6 +448,48 @@ def wait_for_deployment(api_client,
     "Timeout waiting for deployment {0} in namespace {1}".format(
       name, namespace))
 
+
+def wait_for_ingress(api_client,
+                        namespace,
+                        name,
+                        timeout_minutes=2):
+  """Wait for ingress to be ready.
+
+  Args:
+    api_client: K8s api client to use.
+    namespace: The name space for the deployment.
+    name: The name of the ingress.
+    timeout_minutes: Timeout interval in minutes.
+
+  Returns:
+    ingress: The ingress object describing the ingress.
+
+  Raises:
+    TimeoutError: If timeout waiting for ingress to be ready.
+  """
+  # Wait for tiller to be ready
+  end_time = datetime.datetime.now() + datetime.timedelta(
+    minutes=timeout_minutes)
+
+  ext_client_apps = k8s_client.ExtensionsV1beta1Api(api_client)
```

Comment:
ingress should be `networking.k8s.io/v1beta1` in 1.16+. Can you double check API version?

Reply:
Checked Kubernetes 1.16 [ChangeLog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#deprecations-and-removals), so ingress uses `networking.k8s.io/v1beta1` since 1.14, and `extensions/v1beta1` will be removed in v1.20

Reply:
I think it make sense to use other client set to make sure it still works for v1.20+

Reply:
Use `NetworkingV1beta1Api` instead